hudi-utilities/src/test/java/org/apache/hudi/utilities/TestHoodieDeltaStreamer.java

```diff
@@ -394,8 +394,8 @@ private void testUpsertsContinuousMode(HoodieTableType tableType, String tempDir
       } else {
         TestHelpers.assertAtleastNCompactionCommits(5, datasetBasePath, dfs);
       }
-      TestHelpers.assertRecordCount(totalRecords, datasetBasePath + "/*/*.parquet", sqlContext);
-      TestHelpers.assertDistanceCount(totalRecords, datasetBasePath + "/*/*.parquet", sqlContext);
+      TestHelpers.assertRecordCount(totalRecords + 200, datasetBasePath + "/*/*.parquet", sqlContext);
```

Comment:
I could not reason why there is +200 here? The inserts/deletes/updates are calculated to keep the number of records equal to totalRecords.

Reply:
you might be right. as mentioned in the other thread, I am working on the fix. For some reason, my continuous tests times out w/ hitting the expected no of commits.

Reply:
Thanks Sivabalan for such a quick reply.

I had filed https://issues.apache.org/jira/browse/HUDI-667 to work on this fix. You may wish to use it to submit your PR.